SU-HSDO/suhumsci
# SHS-5944: Uninstall Honeypot module

# READY FOR REVIEW

## Summary
- Removes configuration for and uninstalls Honeypot module

## Need Review By (Date)
2024-12-18

## Urgency
low

## Steps to Test
1. Login to the site
2. Visit the modules list page at `/admin/modules`
3. Verify that Honeypot module has been uninstalled

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1209090732666839
